### PR TITLE
Adding CRON with supervisord

### PR DESCRIPTION
**Pull request by tanzaho**
I made a version only for the apache container. Did not add any change to the main and fpm containers.
I added supervisord, and did not try to add Env variables to further configure the cron jobs. This could be added in a future version

### CODE REVIEW

**Comment by tanzaho** [comment]
This references to #4

**Comment by tanzaho** [comment]
I reordered and changed the way cron was started. It should be working now.

File: apache/crontabFile [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+# Every five minutes
+# */5 * * * *
+# Every 30 minutes
+# */30 * * * *
+
+* * * * * /usr/local/bin/php  /var/www/html/app/console mautic:leadlists:update
+* * * * * /usr/local/bin/php /var/www/html/app/console mautic:campaigns:update
+* * * * * /usr/local/bin/php /var/www/html/app/console mautic:campaigns:trigger
+* * * * * /usr/local/bin/php /var/www/html/app/console mautic:email:process 
+* * * * * php /var/www/html/app/console mautic:fetch:email
```
Comment (mbabker) [inline apache/crontabFile]: Can you make the PHP command consistent (either all as `php` or all with the full file path)?

**Comment by milesibastos** [comment]
hi @tanzaho ! u need update the sha1 to bump to 1.3.0:
ENV MAUTIC_SHA1 5d4cc229f860a18ce1b2f29a62e5e0f367d19c0f

**Comment by Gregy** [comment]
Thank you for the pull request. I am sorry but I won't use it because I integrated cron into the container without supervisord (avoiding python dependency).

**Pull request closed without merging**